models/llama.py

```diff
@@ -141,3 +141,5 @@ def get_params_from_model(path):
     serialize(model, args.ov_model_path)
     cost = time.time() - beg
     print(f'serialize done, cost {cost:.2f} seconds.')
+    print(f'save tokenzier to "{os.path.dirname(args.ov_model_path)}" ...')
```

Comment:
Can we create a directory to save model and tokenzier to avoid override the files if multiple models are used?

Reply:
Update args.ov_model_path default value to create model directory to save model and tokenizer, updated README.md